### ISSUE TITLE
can't reproduce the performance of ssd on pascal_voc and coco dataset

### ISSUE THREAD

**Issue by zimenglan-sysu-512** [open-mmlab/mmdetection#3063]
hi 
i run these two commands `./tools/dist_train.sh configs/pascal_voc/ssd300_voc0712.py 4` and `./tools/dist_train.sh configs/ssd/ssd300_coco.py 4`, they both get lower results:  `Epoch [24][5173/5173]  lr: 0.00001, mAP: 0.4037` and `INFO - Epoch [24][18324/18324]        lr: 0.00002, bbox_mAP: 0.1320, bbox_mAP_50: 0.2000, bbox_mAP_75: 0.1440, bbox_mAP_s: 0.0310, bbox_mAP_m: 0.1680, bbox_mAP_l: 0.2450, bbox_mAP_copypaste: 0.132 0.200 0.144 0.031 0.168 0.245`. 
any help is appreciated.
thanks

**Comment by yhcao6** [comment]
It seems you don't use default training schedule. Please adjust the learning accordingly following the tutorial https://github.com/open-mmlab/mmdetection/blob/master/docs/getting_started.md#train-a-model

**Comment by zimenglan-sysu-512** [comment]
@yhcao6 
i adjust the learning rate accordingly following the turorial, and the results still get lower performance.

**Comment by zimenglan-sysu-512** [comment]
hi @yhcao6 
i try difference learning rate, i still get lower results, e.g. for the first epoch, it gets 
```
+-------------+------+-------+--------+-------+                                                                                                                                                
| class       | gts  | dets  | recall | ap    |                                                                                                                                                
+-------------+------+-------+--------+-------+                                                                                                                                                
| aeroplane   | 285  | 3906  | 0.775  | 0.417 |                                                                                                                                                
| bicycle     | 337  | 3385  | 0.780  | 0.394 |                                                                                                                                                
| bird        | 459  | 8419  | 0.732  | 0.329 |                                                                                                                                                
| boat        | 263  | 13969 | 0.681  | 0.270 |                                                
| bottle      | 469  | 11674 | 0.478  | 0.157 |                                                                                                                                                
| bus         | 213  | 3404  | 0.826  | 0.456 | 
| car         | 1201 | 18744 | 0.821  | 0.410 |                                                                                                                                                
| cat         | 358  | 4398  | 0.877  | 0.461 |                                                                                                                                                
| chair       | 756  | 53088 | 0.692  | 0.176 |                                                                                                                                                
| cow         | 244  | 4036  | 0.832  | 0.369 |                                                                                                                                                
| diningtable | 206  | 4543  | 0.675  | 0.263 |             
| dog         | 489  | 8491  | 0.892  | 0.441 |                                                                                                                                                
| horse       | 348  | 3600  | 0.848  | 0.425 |                                                                                                                                                
| motorbike   | 325  | 3045  | 0.763  | 0.441 |                                                                                                                                                
| person      | 4528 | 80395 | 0.741  | 0.333 |                                                                                                                                                
| pottedplant | 480  | 28741 | 0.602  | 0.152 |                                                                                                                                                
| sheep       | 242  | 5179  | 0.764  | 0.327 |             
| sofa        | 239  | 2994  | 0.720  | 0.322 |                                                                                                                                                
| train       | 282  | 8170  | 0.894  | 0.565 | 
| tvmonitor   | 308  | 9754  | 0.792  | 0.395 |                                                                                                                                                
+-------------+------+-------+--------+-------+                                                                                                                                                
| mAP         |      |       |        | 0.355 |                                                                                                                                                
+-------------+------+-------+--------+-------+ 
```
 can u give me some tips to debug what is going wrong?
thanks.

**Comment by yhcao6** [comment]
There is something wrong with SSD VOC on mmdet v2.x. But SSD COCO should be OK, this is the training log: 
https://open-mmlab.s3.ap-northeast-2.amazonaws.com/mmdetection/v2.0/ssd/ssd300_coco/ssd300_coco_20200307_174216.log.json.

Could you describe your training setting: like how many GPUs you use and the learning rate you set?

**Comment by zimenglan-sysu-512** [comment]
hi @yhcao6 
i use the newest code to train ssd right now.
for pascal, i `only` modify the learning rate in `configs/pascal_voc/ssd300_voc0712.py`, use 4 gpus to train this config. my env: torch==1.4.0 torchvision==0.5.0.
for coco, i will run `configs/ssd/ssd300_coco.py` again to see what happen.
thanks.

**Comment by yhcao6** [comment]
I have trained SSD VOC some times ago and the performance is slightly lower than mmdet1.x. Once I found where the problem is I will update the benchmark.
I will use 4 GPU to train SSD VOC so that we can compare the results.

**Comment by zimenglan-sysu-512** [comment]
hi @yhcao6 
the result of `ssd300_coco`:
```
Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.128
Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.193
Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.140
Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.031
Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.164
Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.238
Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.195
Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.276
Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.295
Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.044
Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.302
Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.526
```

**Comment by yhcao6** [comment]
This is my training log. Train ssd300 pascal voc with 4GPU, I only change lr to 5e-4. I think you get something wrong. 
[20200624_135247.log](https://github.com/open-mmlab/mmdetection/files/4832404/20200624_135247.log)
```
+-------------+------+-------+--------+-------+
| class       | gts  | dets  | recall | ap    |
+-------------+------+-------+--------+-------+
| aeroplane   | 285  | 1850  | 0.902  | 0.788 |
| bicycle     | 337  | 1700  | 0.920  | 0.830 |
| bird        | 459  | 4539  | 0.887  | 0.746 |
| boat        | 263  | 5259  | 0.897  | 0.703 |
| bottle      | 469  | 8986  | 0.785  | 0.479 |
| bus         | 213  | 1334  | 0.962  | 0.849 |
| car         | 1201 | 8866  | 0.955  | 0.856 |
| cat         | 358  | 1293  | 0.955  | 0.879 |
| chair       | 756  | 16017 | 0.882  | 0.590 |
| cow         | 244  | 1334  | 0.959  | 0.809 |
| diningtable | 206  | 1865  | 0.908  | 0.726 |
| dog         | 489  | 2069  | 0.949  | 0.852 |
| horse       | 348  | 1276  | 0.937  | 0.857 |
| motorbike   | 325  | 1335  | 0.926  | 0.833 |
| person      | 4528 | 37629 | 0.920  | 0.782 |
| pottedplant | 480  | 9549  | 0.794  | 0.494 |
| sheep       | 242  | 1809  | 0.921  | 0.753 |
| sofa        | 239  | 1310  | 0.937  | 0.785 |
| train       | 282  | 1359  | 0.950  | 0.881 |
| tvmonitor   | 308  | 3502  | 0.922  | 0.764 |
+-------------+------+-------+--------+-------+
| mAP         |      |       |        | 0.763 |
+-------------+------+-------+--------+-------+
```

**Comment by zimenglan-sysu-512** [comment]
hi @yhcao6 
u are right, i reinstall the `envs` and pull the newest code to train again. this time the results look normal.  thanks a lot.

**Comment by liuzhi111** [comment]
if i try to use 2 GPUs，i only have to change the lr to 2.5e-4?Please help me?Thanks!Because I can't reproduce the accuracy like you.
